### DISCUSSION TITLE
PrimeReact update

### DISCUSSION THREAD

**Discussion by vulturana1** [primefaces/community#3208] [PrimeReact]
When will be PrimeReact updated to be in sync with PrimeNG v18? Will be the theming reimplemented also in PrimeReact to also use APIs like in PrimeNG v18, or it will continue to use the theme.css?

**Comment by melloware** [comment]
As far as I know PrimeReact 11 is currently under Heavy development to bring it in line with PrimeNG and PrimeVue.

**Comment by hale** [comment]
Does anyone know of an issue we can follow to track that? I couldn't see anything in [the 11.0.0 milestone](https://github.com/primefaces/primereact/milestone/92) - thanks!

**Reply by melloware** [reply]
No PrimeTek tracks it internally...
